### PR TITLE
feat: FLUI-67 72

### PR DESCRIPTION
**Pull request by atoulous**
# FIX : antd updates

- closes #TICKET_NUMBER

## Description
after antd update to 4.24.10, few things should be fix
a new class ant-space-compact-block breaks our dropdown.button line

[JIRA LINK]

Acceptance Criterias

## Validation

- [ ] Storybook add or modified
- [ ] version Update in package.json and Release.md
- [ ] Code Approved
- [ ] QA Done
- [ ] Design/UI Approved from design

### After
image

## QA

Steps to validate
Url (storybook, ...)
...

### CODE REVIEW

File: packages/ui/src/utils/localeUtils.test.ts
Comment (AltefrohneGaelle) [inline packages/ui/src/utils/localeUtils.test.ts:1]: Pourquoi avoir supprimer le fichier de test ?

Comment (atoulous) [reply packages/ui/src/utils/localeUtils.test.ts:1]: une erreur

File: packages/ui/src/components/QueryBuilder/Header/Tools/SavedFiltersMenu.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ const SavedFiltersMenu = ({
     const { dictionary } = useContext(QueryBuilderContext);
     const [manageFiltersVisible, setManageFiltersVisible] = useState(false);
 
-    const confirmUnsavedChangeForExistingFilter = (onOkCallback: Function) => {
+    const confirmUnsavedChangeForExistingFilter = (onOkCallback: any) => {
```
Comment (AltefrohneGaelle) [inline packages/ui/src/components/QueryBuilder/Header/Tools/SavedFiltersMenu.tsx:35]: On pourrait typer void non ?
```suggestion
    const confirmUnsavedChangeForExistingFilter = (onOkCallback: () => void) => {
```

**Comment by AltefrohneGaelle** [comment]
et je pense il manque l'update de version ou tu veux surcharger la derniere ?

**Pull request merged**